src/main/java/data/TimeSpan.java

```diff
@@ -5,7 +5,15 @@
  */
 public class TimeSpan implements Comparable<TimeSpan> {
     
-    //TODO Replace magic numbers with constants
+    public static final int MIN_HOUR = 0;
+    public static final int MIN_MINUTE = 0;
+    public static final int MAX_MINUTE = 59;
```

Comment:
Are these constant values public for a specific reason?

Reply:
Yes, so that the user of this class knows what values the constructor is allowed to call with.

Reply:
That's fine! It just wasn't clear for me at the first moment but I see what your intention was :)